doc/rainbow-delimiters.txt

```diff
@@ -145,9 +145,9 @@ Alternatively, the same configuration in Lua:
                                                     *rainbow-delimiters.setup*
 'rainbow-delimiters.setup'.setup({config})
```

Comment:
Same comment as below. Maybe this can be `'rainbow-delimiters'.setup({config})`?